### PR TITLE
Accordion

### LINKED ISSUE #13: Build accordion component
Spec: https://github.com/ubuntudesign/vanilla-dashboard-theme-design/tree/master/Accordion

### PR DESCRIPTION
**Pull request by richmccartney**
## Done

- Creates and accessible accordion pattern to be used in the dashboard theme.

## QA

- Pull down branch and run `./run` 
- View the new pattern here http://0.0.0.0:8102/vanilla-dashboard-theme/examples/patterns/accordion/accordion/
- Check the designs here https://github.com/ubuntudesign/vanilla-dashboard-theme-design/blob/master/Accordion

## Details

- Fixes #13

### CODE REVIEW

**Comment by Lukewh** [comment]
## QA

On first load the accordion buttons transition from a grey background to white. Just need the `transition-duration: 0s` to sort that out.

File: scss/_patterns_accordion.scss [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    &__tab {
+      background: {
+        position: top 1.25rem right $sp-medium;
+        repeat: no-repeat;
+      }
+      border: 0;
+      padding: $sp-medium $sp-xx-large $sp-medium $sp-small;
+      text-align: left;
+      width: 100%;
```
Comment (Lukewh) [inline scss/_patterns_accordion.scss]: `transition-duration: 0s;` here to sort the flash of grey.

**Comment by richmccartney** [comment]
@Lukewh Can you give it a second look please :)

**Comment by Lukewh** [comment]
@richmccartney will take a look when you've sorted the lint error

**Pull request merged**